### ISSUE TITLE
Copilot doesn't provide ability to enable cloudwatch performance monitoring of cluster.

### ISSUE THREAD

**Issue by sanjeevtejyan** [aws/copilot-cli#1579]
To enable cloudwatch for copilot rolled-out cluster, one has to execute aws command manually to enable the cloudwatch to monitor the performance of individual task and service. But it can be made part of configuration when a ECS cluster is setup.

**Comment by efekarakus** [comment]
Heya @sanjeevtejyan !

To make sure that I understand the request correctly, do you mean you're looking for a way of enabling [ECS CloudWatch Container Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-metrics-ECS.html) with Copilot? Or are you asking about different metrics?

Thanks!

**Comment by sanjeevtejyan** [comment]
yes that is correct. Can we hook any other metric to measure the performance of the cluster?

**Comment by tam0ri** [comment]
Similar issue: #1614

Currently, `AWS::ECS::Cluster` resource does not have `ClusterSettings` property in a template.
https://github.com/aws/copilot-cli/blob/069bcf00d7edd8899db939963084ebd8b4767b01/templates/environment/versions/cf-v1.3.0.yml#L49-L57

So Container Insights setting depends on account level settings. I think it's useful if we can enable/disable when creates environment.

## Proposal

Adding an option for `env init`.

### Option 1:
```
--container-insights enabled | disabled
```

When the option is not specified, `ClusterSettings` property is removed from template and depends on account settings.


### Option 2:
```
--container-insights-enabled true | false
```

When the option is not specified, set true or false as default value.

**Comment by dannysteenman** [comment]
+1 for this feature!

> Similar issue: #1614
> 
> Currently, `AWS::ECS::Cluster` resource does not have `ClusterSettings` property in a template.
> 
> https://github.com/aws/copilot-cli/blob/069bcf00d7edd8899db939963084ebd8b4767b01/templates/environment/versions/cf-v1.3.0.yml#L49-L57
> 
> So Container Insights setting depends on account level settings. I think it's useful if we can enable/disable when creates environment.
> 
> ## Proposal
> Adding an option for `env init`.
> 
> ### Option 1:
> ```
> --container-insights enabled | disabled
> ```
> 
> When the option is not specified, `ClusterSettings` property is removed from template and depends on account settings.
> 
> ### Option 2:
> ```
> --container-insights-enabled true | false
> ```
> 
> When the option is not specified, set true or false as default value.

**Comment by dannysteenman** [comment]
+1 for this feature!

**Comment by efekarakus** [comment]
Hi folks! @sanjeevtejyan, @flemzord, @git823, @dannysteenman  

Out of curiosity, for environments where you'd like to enable container insights, would you also want ELB logging (#1530) for the load balancer and VPC flow logs for observability?
What would y'all think of an approach where running `copilot env init --prod` by default enabled all these observability settings instead of specifying them individually?

**Comment by huanjani** [comment]
This is now released 🎉 in v1.14.0 https://github.com/aws/copilot-cli/releases/tag/v1.14.0 !
